### ISSUE TITLE
examine relationship to PhenX measures and LOINC parts

### ISSUE THREAD

**Issue by mellybelly** [monarch-initiative/MAxO#59]
e.g. https://www.phenxtoolkit.org/resources/download/
https://loinc.org/articles/?id=364
https://loinc.org/news/loinc-version-2-65-and-relma-version-6-24-are-now-available/

@djvreeman maybe you can point us at other resources and help us understand how our efforts on this ontology might relate. Don't want to duplicate effort!

**Comment by LCCarmody** [comment]
Naive glance, I don't see a lot of overlap with phenxtoolkit, but would definitely welcome other input.
Thanks.

**Comment by mellybelly** [comment]
there are phenX "answers" in LOINC - this is why i want us to understand this space :-)

**Comment by pnrobinson** [comment]
I agree it would be useful to explore phenX!

**Comment by djvreeman** [comment]
Yes, I can explain. Back in 2010ish, we (LOINC) launched a collaboration with the PhenX team at RTI led by Carol Hamilton whereby nearly all of the PhenX content was represented in LOINC.

PhenX mostly organizes its content into Domains (e.g. Anthropometrics, Cancer, Cardiovascular) and Protocols (e.g. Hand Dominance by Edinburgh Handedness Inventory vs Hand Dominance by Avon Longitudinal Study of Parents and Children Protocol (ALSPAC)).

We represent these collections of Domains and Protocols as "Panels" in LOINC. That is, enumerated collections of terms in parent/child relationship.

For example, the whole set of content in the Anthropometrics domain is identified by the LOINC panel term _PhenX domain - Anthropometrics_ LN:[61130-1](https://lo.inc/61130-1), the collection of ALSPAC items is LN:[56096-1](https://lo.inc/56096-1), and one of the questions in the ALSPAC is, for example, asking about hand dominance when throwing a ball LN:[56098-7](https://lo.inc/56098-7). 

For questions that have ordinal or nominal responses, we create LOINC Answer Lists (identified by LL* codes) and enumerate the possible answer choices as LOINC Answers (identified by LA* codes).

After our initial work several years ago (which was funded in a bolus), we did not keep synchronized with the PhenX team, so some additional content was added/retired in PhenX but not represented in LOINC. We presently have a grant subcontract with RTI where we are focused on refreshing the content related to environmental exposures, so are doing some updating. But it remains a work in progress.

In our soon-to-be-published LOINC 2.66 version, we'll have an updated way of searching in LOINC by PhenX ID to find the matching LOINC term.

Future work might be to publish those mappings as a FHIR ConceptMap resource in the LOINC API.
